### PR TITLE
net/haproxy: add syslog-ng socket

### PR DESCRIPTION
**Pull request by kulikov-a**
Hi
for discussion on https://github.com/opnsense/plugins/issues/2587#issuecomment-954598043
drop 'local'
add log socket
if i'm not mistaken this can be applied as a patch without waiting for the https://github.com/opnsense/core/commit/a763313d138258b0044c3285de0e8ac55d308160
Thanks!

### CODE REVIEW

File: net/haproxy/src/etc/inc/plugins.inc.d/haproxy.inc [hunk 1]
```diff
@@ -37,8 +37,7 @@ function haproxy_syslog()
     $syslogconf = array();
 
     $syslogconf['haproxy'] = array(
-        'local' => '/var/haproxy/var/run/log',
-        'facility' => array('haproxy'),
+        'facility' => array('haproxy')
```
Comment (fichtner) [inline net/haproxy/src/etc/inc/plugins.inc.d/haproxy.inc:40]:
```suggestion
        'facility' => array('haproxy'),
```

trailing commas are nice, they reduce diff size

Comment (kulikov-a) [reply net/haproxy/src/etc/inc/plugins.inc.d/haproxy.inc:40]: sorry for that)

**Comment by fraenki** [comment]
I've tested it on my test machine and HAProxy logging is still working as expected. However, it was already working before applying the patch, so I'm probably not the ideal test candidate. :)

I'd merge the PR, but it is marked as draft. Is there anything missing?

**Comment by kulikov-a** [comment]
> it was already working before applying the patch, so I'm probably not the ideal test candidate. :)

with clog disabled?

> I'd merge the PR, but it is marked as draft. Is there anything missing?
its ready. I just don't know your plans. @AdSchellevis  has already merged https://github.com/opnsense/core/commit/eab536021e742c0f3c48727613b178f37d3c1cd5 and I'm not sure if an empty string is still needed in the socket template

**Comment by fraenki** [comment]
OK, successfully tested again with clog disabled.

**Comment by fraenki** [comment]
@kulikov-a Thanks!

**Comment by kulikov-a** [comment]
Thanks!

**Pull request merged**